### ISSUE TITLE
loading  job slowly

### ISSUE THREAD

**Issue by bruse-peng** [apache/shardingsphere-elasticjob#2196]
## Question

**For English only**, other languages will not accept.

 when i configure the zk address and namespace and connet it, it loading data slowly ,it takes about 10 mintunes loading and show the data, how should i loading data fast

Before asking a question, make sure you have:

- Googled your question.
- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere-elasticjob/issues).
- Read documentation: [ElasticJob Doc](https://shardingsphere.apache.org/elasticjob/current/en/overview/).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

**Comment by TeslaCN** [comment]
Do you have many jobs in registry center?

**Comment by SuperCarrys** [comment]
https://github.com/apache/shardingsphere-elasticjob/issues/2189

You can see if the problem described in this link is similar to the situation you encountered, and there is a solution to the problem at the end

**Comment by bruse-peng** [comment]
> Do you have many jobs in registry center?

yes our application in kubernets,when i restart pod the ip will change .and all history pods ip will record in zookeeper /servers  how should i delete it
